gdc_client/download/parser.py

```diff
@@ -41,7 +41,6 @@ def get_client(args, index_client):
         'directory': args.dir,
         'segment_md5sums': args.segment_md5sums,
         'file_md5sum': args.file_md5sum,
-        'debug': logging.DEBUG in args.log_levels,
```

Comment:
Why remove debug?

Reply:
I think these flags are handled by the logger.